core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -63,6 +64,29 @@ class TopicConfigHandler(private val logManager: LogManager, kafkaConfig: KafkaC
       val logConfig = LogConfig(props)
       logs.foreach(_.config = logConfig)
     }
+
+    val brokerId: Int = kafkaConfig.brokerId
+
+    if (topicConfig.containsKey(LogConfig.ThrottledReplicasListProp)) {
+      val partitions = parseThrottledPartitions(topicConfig, brokerId)
+      quotas.leaderReplication.markReplicasAsThrottled(topic, partitions)
+      quotas.followerReplication.markReplicasAsThrottled(topic, partitions)
+      logger.info(s"Setting throttled partitions on broker $brokerId to ${partitions.map(_.toString)}")
```

Comment:
It seems to me that the `partitions.map(_.toString)` doesn't do anything since the default behaviour is to call `toString` on each element anyway.